### DISCUSSION TITLE
Config option file paths with docker-compose

### DISCUSSION THREAD

**Discussion by AlexanderRitter02** [ArchiveBox/ArchiveBox#779] [Q&A] [answered]
Hi,
I use ArchiveBox with docker-compose under Windows.
Currently, I'm wondering how do I pass files (like for example `cookies.txt` or the chrome data dir) into the config?

Putting the path in normally doesn't seem to work and I didn't find anything in the documentation.
In another post I read something about _mounting_, however I have no clue why and how that'd be necessary.

I'd be grateful for a short explanation/guide, thanks! :heart:

Perhaps an example would be great too:
- Suppose I have the file `cookies.txt` in the `data` folder (besides the ArchiveBox.config)
- What would be the steps and what path would I need to put for COOKIES_FILE to make it work

**Comment by pirate** [comment, marked as answer]
If `cookies.txt` is in the `data` folder, the the path should be `COOKIES_TXT=/data/cookies.txt`.

(`data` is in the hard drive root inside docker, so the paths are very easy)
